src/stats.jl

```diff
@@ -119,28 +109,28 @@ function changerate(
     names_of_params = names(_chains)
 
     if append_chains
-        df = chaindataframe_changerate("Change Rate", names_of_params, _chains.value.data)
-        return df
+        stats = summarystats_changerate("Change Rate", names_of_params, _chains.value.data)
+        return stats
     else
-        vector_of_df = [
-            chaindataframe_changerate(
+        vector_of_stats = [
+            summarystats_changerate(
                 "Change Rate - Chain $i", names_of_params, data
             )
             for (i, data) in enumerate(to_vector_of_matrices(_chains))
         ]
-        return vector_of_df
+        return vector_of_stats
     end
 end
 
-function chaindataframe_changerate(name, names_of_params, chains; kwargs...)
+function summarystats_changerate(name, names_of_params, chains; kwargs...)
     # Compute the change rates.
     changerates, mvchangerate = changerate(chains)
 
     # Summarize the results in a named tuple.
-    nt = (; zip(names_of_params, changerates)..., multivariate = mvchangerate)
```

Comment:
Lacking a `parameter` column meant the `show` method was broken. But since there is a changerate for every parameter, it makes more sense to do the same thing as `gelmandiag_multivariate` and return a `SummaryStats` for the marginal values and return the multivariate changerate separately.

Reply:
Genuine question: what is the change-rate in this context?

Reply:
From inspecting the code, it's, for each parameter and chain, the fraction of draws that are different from the previous draw. I suppose it's similar to "acceptance rate."